app/routes/api.js

```diff
@@ -39,6 +39,47 @@ router.get('/fact', (req, res) => {
     }
 });
 
+// API endpoint to add a new fact to the database.
+router.post('/fact', (req, res) => {
+    res.setHeader('Content-Type', 'application/json');
```

Comment:
You don't need to set the header to 'app/json' if you use res.json() (http://restify.com/docs/response-api/#json).